app/admin-tab/host/template.hbs

```diff
@@ -1,37 +1,21 @@
 <section class="header">
   <ol class="breadcrumb right-divider">
-    <li class="active">Host Registration</li>
+    <li class="active">Settings</li>
   </ol>
 </section>
 
 <section>
   {{top-errors errors=errors}}
-  <h5>We need to know a little about how your environment is set up before you can register hosts.</h5>
 </section>
 
 <section class="well">
-  <h5>What base URL should hosts use to connect to the Rancher API?</h5>
-
-  <div style="margin-top: 15px;">
-    <label>{{radio-button selection=customRadio value="no"}}&nbsp;This site's address:&nbsp;&nbsp;</label>
-    <code class="form-control-static">{{thisPage}}</code>
-  </div>
-
-  <div style="margin-top: 10px;">
-    <label>{{radio-button selection=customRadio value="yes"}}&nbsp;Something else:&nbsp;&nbsp;</label>
-    {{input type="text" class="form-control" value=customValue placeholder="e.g. rancher.mydomain.com" safeStyle="width: calc(100% - 150px); display: inline-block;  min-width: 300px;"}}
-    <h6 style="margin-top: 10px;">Don't include <code>/v1</code> or any other path, but if you are doing <a href="http://docs.rancher.com/rancher/installing-rancher/installing-server/basic-ssl-config/" target="_blank">SSL termination</a> in front of Rancher, be sure to use <code>https://</code>.</h6>
-  </div>
+  {{host-settings customValue=customHostValue custom=custom thisPage=thisPage customRadio=customRadio sendActiveValue="setActiveValue"}}
```

Comment:
custom/thisPage/customRadio can move into the component.  sendActiveValue can be just `(action mut model.host)`